Added 10 questions

Shemozzle opened the pull request:
10 questions about 310/510 are added.

Commit:
Added 10 questions

10 questions about 310/510 are added.

h3x reviewed (changes requested):
you dont have to write question 1, question 2 etc, 
just like this 

"question" : "Planning is the intelligent estimate of resources required to perform a predefined project successfully at a future date within a defined environment.",
"a": "True",
"b": "False",
"c": "NA",
"d": "NA",
"answer": "a"

and anser can only be a,b,c, or d. putting T or F in there wont work

also formatting is fucked up a bit

Commit:
Reformatted the questions

Removed some T/F questions and replaced them. Also adjusted the formatting of the question set.
